docs/source/en/quantization/overview.md

```diff
@@ -72,4 +72,10 @@ We value your feedback to help identify bugs before the full release! Check out
 
 \** bitsandbytes is seeking contributors to help develop and lead the Apple Silicon backend. Interested? Contact them directly via their repo. Stipends may be available through sponsorships.
 
-</Tip>
\ No newline at end of file
+</Tip>
+
+<Tip>
+
+\*** GPTQ only supports 4-bit on Intel CPU / GPU.
+
+</Tip>
```

Comment:
Let's move that to the docs related to gptq

Reply:
@SunMarc  I have updated the table and fixed most issues and reformatted so everything is consistent. BitsAndBytes and Torchao had non-standard value formatting for the table. The footnotes are now standardized 1,2,3 subscripts and not the current *** method.